Romero tellez javier eduardo

JavierEduardoRomeroTellez opened the pull request:
Se agrego la carpeta Javier-Eduardo-Romero-Tellez y dentro de la misma se creo un README con información del programa

Se agrego un programa en python de un arbol binario de n valores.

JavierEduardoRomeroTellez committed:
Create README.md

Se creo y modifico el README

JavierEduardoRomeroTellez committed:
Modificacion del python

Ahora el usuario puede elegir cuantos datos ingresar al arbol binario

tri-rsa reviewed (approved):
Esta bien, pero en vez de que pida dato por dato, esto se puede asignar aleatoriamente.
Esta OK

JavierEduardoRomeroTellez committed:
Cambio sugerido realizado

Se agrego la libreria random para asi generar datos aleatorios que se insertaran en el arbol
